tembo-operator/tests/integration_tests.rs

```diff
@@ -2958,7 +2958,11 @@ mod test {
                         "routing": [
                             {
                                 "port": 3000,
-                                "ingressPath": "/"
+                                "ingressPath": "/",
+                                "entryPoints": [
+                                    "websecure"
+                                ],
```

Comment:
existing app_services do not need to start including a value for `entryPoints` right? Let's leave the test using the default since there's already services that exist that assume it will default to `websecure`